### PR TITLE
Confusing --output overwrite (append) behaviour

### PR DESCRIPTION
**Pull request by oversoul**
Changed the first tests output to match the server output, for some reason it doesn't seem to detect the `mailto`.

sorry if this is a mess.

### CODE REVIEW

File: tests/ScanCommandTest.php [hunk 1]
```diff
@@ -96,7 +96,8 @@ protected function appearsInConsoleOutput($texts)
     }
 
     /** @test */
-    public function it_removes_output_file_on_overrite() {
+    public function it_removes_output_file_on_overrite()
```
Comment (freekmurze) [inline tests/ScanCommandTest.php]: Please correct the spelling. Change `overrite` to `overwrite` everywhere in this package.

File: tests/ScanCommandTest.php [hunk 2]
```diff
@@ -107,17 +108,17 @@ public function it_removes_output_file_on_overrite() {
         $commandTester->setInputs(['y']);
         $commandTester->execute([
             'url'      =>  'http://localhost:8080',
-            '--output' =>  $this->overriteFile
+            '--output' =>  $this->overriteFile,
         ]);
 
-        $this->assertfileequals(__dir__.'/fixtures/overrite.txt', $this->overriteFile);
+        $this->assertfileequals(__DIR__.'/fixtures/overrite.txt', $this->overriteFile);
```
Comment (freekmurze) [inline tests/ScanCommandTest.php]: Use proper casing: `assertFileEquals`

**Comment by freekmurze** [comment]
Thanks!

**Comment by oversoul** [comment]
You are welcome!

**Pull request merged**